jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/FilePropertiesStack.java

```diff
@@ -34,10 +34,23 @@
   // TODO perhaps use a fixed size list here
   private final List<FilePropertiesSpec> stack = new ArrayList<>(3);
 
-  @Nullable private FilePermissions filePermissions;
-  @Nullable private FilePermissions directoryPermissions;
-  @Nullable private Instant modificationTime;
-  @Nullable private String ownership;
+  private FilePermissions filePermissions;
+  private FilePermissions directoryPermissions;
+  private Instant modificationTime;
+  private String ownership;
+
+  /** Create new FilePropertiesStack with defaults. */
+  public FilePropertiesStack() {
+    setDefaults();
+  }
+
+  private void setDefaults() {
+    filePermissions = FilePermissions.DEFAULT_FILE_PERMISSIONS;
+    directoryPermissions = FilePermissions.DEFAULT_FOLDER_PERMISSIONS;
+    modificationTime = FileEntriesLayer.DEFAULT_MODIFICATION_TIME;
+    // TODO: get default from FileEntriesLayer (requires buildplan release)
```

Comment:
Do we need another release? 0.5.0? I don't recall new changes since 0.4.0.

Reply:
I mean if we were to define a default in the build plan, this would be required.

Reply:
The thing is, I think there might be some other minor additions to the buildplan that might come up, so we can hold off on this.